src/libmerklecpp/test/demo_tree.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+
+#ifdef HAVE_EVERCRYPT
+#  include <MerkleTree.h>
+#endif
+
+#include "util.h"
+
+#include <merklecpp.h>
+
+#define PRINT_HASH_SIZE 3
+
+int main()
+{
+  try
+  {
+    const size_t num_leaves = 6;
+    // for (size_t num_leaves = 1; num_leaves < 121; num_leaves++)
```

Comment:
TDB?

Reply:
To select how much and which ones you want :)